authortechnical.Rmd

```diff
@@ -296,4 +318,5 @@ knitr::include_graphics("images/hugochecks.png")
 knitr::include_graphics("images/hugocheckspassed.png")
 ```
 
-
+[^1]: In Hugo speak, we'd say your post is a [leaf bundle](https://gohugo.io/content-management/page-bundles/#leaf-bundles).
+[^2]: If you don't use RStudio you can still use the addin, but the new post will be opened in [`getOption("editor")`](https://stat.ethz.ch/R-manual/R-devel/library/base/html/options.html).
```

Comment:
I don't quite understand this "the new post will be opened in [`getOption("editor")`](https://stat.ethz.ch/R-manual/R-devel/library/base/html/options.html)"

Does the user need to set their preferred editor that way?

If yes, maybe rephrase